### DISCUSSION TITLE
关于官网的Pro

### DISCUSSION THREAD

**Discussion by hxguo** [dream-num/univer#1741] [Q&A | 常见问题] [answered]
在官网有看到Pro版本，[https://univer.ai/pro/zh-cn/enterprises/](https://univer.ai/pro/zh-cn/enterprises/)
想问一下Pro版本是什么样的收费模式？
文档中没有留下联系方式，discord又进不去。只能在这里问一下

**Comment by jikkai** [comment, marked as answer]
中文用户可以在这里找到其他的社区入口：
https://univer.ai/zh-cn/guides/introduction/#%E4%B8%8B%E4%B8%80%E6%AD%A5

**Reply by hxguo** [reply]
多谢
